### PR TITLE
Added tagging and favorites page.

### PR DESCRIPTION
**Pull request by susodapop**
This is my first post that includes images.

### CODE REVIEW

File: website/_kb/user-guide/querying/favorites-tagging.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+category: querying
```
Comment (arikfr) [inline website/_kb/user-guide/querying/favorites-tagging.md]: This is relevant for dashboards too. Do you want to adapt the content to work for both? 

We will also need to figure how to place the same page in two categories (currently not supported).

File: website/_kb/user-guide/querying/favorites-tagging.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: Favorites & Tagging
+slug: favorites-tagging
+---
+
+Redash users write a lot of queries and dashboards! Favorites and Tagging are here to make finding them easy as your list of queries grows from a few hundred to a few thousand.
+
+<img src="/assets/images/docs/gitbook/favorites-example.png" width="100%">
+
+## Favorites
+You can favorite a dashboard or query by clicking the star to the left of its title anywhere in Redash. The star will turn gold to indicate success. Your list of favorites of each kind can be viewed from the query or dashboard list view. Just click the `Favorites` filter on the right.
```
Comment (arikfr) [inline website/_kb/user-guide/querying/favorites-tagging.md]: It's yellow, not gold. :)

Comment (arikfr) [reply website/_kb/user-guide/querying/favorites-tagging.md]: We should mention that favorites are also available from the homepage and navbar dropdowns.

File: website/_kb/user-guide/querying/favorites-tagging.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+
+## Tagging
+You can tag queries by subject matter, location, user or any parameter that is meaningful to your organization. Tags are added from the query editor or the dashboard editor. Hover your mouse on the query or dashboard title and an `+Add Tag` button will appear. In the modal that appears you can select as many tags as you need. The modal will suggest previously-used tags as you type. Hit `Save` when you're finished or `Esc` to abort tagging.
```
Comment (arikfr) [inline website/_kb/user-guide/querying/favorites-tagging.md]: Worth adding a callout mentioning that it's good idea to have an internal discussion on the taxonomy they want to create with tags.

File: website/_kb/user-guide/querying/favorites-tagging.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+<img src="/assets/images/docs/gitbook/tagging-example.png" width="100%">
+
+Your tags will appear on the Dashboard and Query list views on the righthand side. Click on any tag to filter the list view instantly. Click a second time to remove the filter.
```
Comment (arikfr) [inline website/_kb/user-guide/querying/favorites-tagging.md]: They can also Shift + Click to select multiple tags.

File: website/_kb/user-guide/querying/favorites-tagging.md (hunk 1 above)
Comment (susodapop) [reply website/_kb/user-guide/querying/favorites-tagging.md]: I did this the simplest way possible: see PR #161

**Comment by susodapop** [comment]
I won't merge prior to review again.

**Pull request merged**